Tutorials/G-CNN_Honeycomb.ipynb

```diff
@@ -0,0 +1,812 @@
+{
```

Comment:
<p>Maybe say something like "on my CPU ..." or add the approximate CPU/GPU models you've used for these timings as naturally performance will differ a bit between machines (e.g., on my CPU this is more like 8 s/it), though the orders of magnitude should probably remain for most setups.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/netket/netket/pull/647/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>This a good point, I’ll try to make it more clear</p>